concepts/measurements/ventsettings.sql

```diff
@@ -0,0 +1,141 @@
+DROP TABLE IF EXISTS `physionet-data.mimic_derived.ventsettings`;
+CREATE TABLE `physionet-data.mimic_derived.ventsettings` AS
```

Comment:
delete this, also I am avoiding plural so "measurements" -> "measurement" and "ventsettings" to "ventsetting". I actually think this table will end up named "ventilator_setting".

I'm also in the middle of overhauling the ventilation code with a colleague so don't put too much effort in here..

Reply:
Will do. But is it alright to use the `ventsetting` table for now? (Just so I can move along my project.)

Reply:
yes but I've renamed it `ventilator_setting` and some of the columns are renamed - let me know if it's missing anything